pkg/sfu/connectionquality/connectionstats.go

```diff
@@ -81,18 +81,42 @@ func (cs *ConnectionStats) OnStatsUpdate(fn func(cs *ConnectionStats, stat *live
 }
 
 func (cs *ConnectionStats) UpdateMute(isMuted bool, at time.Time) {
+	/* TODO-RESTORE
+	if cs.done.IsBroken() {
+		return
+	}
+	*/
```

Comment:
want to add these to protect against adding stuff to scorer after close. But, commenting out for now to check what happens with leaks. Will restore these after collecting some data.